assets/js/modules/adsense/util/parsing.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Parsing utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { isValidAccountID, isValidClientID } from './validation';
+
+/**
```

Comment:
It would be good to add some unit tests for these functions; they look straightforward but it'd be good to test them and make sure we're covering any edge cases, unexpected inputs, etc. 😄